src/amp-story-player/amp-story-player-iframe-pool.js

```diff
@@ -77,4 +77,58 @@ export class IframePool {
 
     return detachedStoryIdx;
   }
+
+  /**
+   * Clears stored story indices.
+   * @return {!Array<number>}
+   */
+  evictStories() {
+    const evictedStories = this.storyIdsWithIframe_;
+    this.storyIdsWithIframe_ = [];
+    return evictedStories;
+  }
+
+  /**
+   * @return {!Array<number>}
+   */
+  getAvailableIframeIdx() {
+    return this.iframePool_;
+  }
+
+  /**
+   * Finds adjacent iframe indices given an index.
+   * Examples (x is provided index):
+   * [x] [1] [2] [] []
+   * [2] [x] [1] [] []
+   * [] [] [2] [1] [x]
+   * [1] [x]
+   * @param {!Array<number>} queue
+   * @param {!Array<number>} adjacent
+   * @param {number} numStories
+   */
+  findAdjacent(queue, adjacent, numStories) {
```

Comment:
`@private`?

Reply:
These are used in `amp-story-player-impl.js` so they are not private.